geomstats/geometry/beta_distributions.py

```diff
@@ -292,6 +292,13 @@ def exp(self, tangent_vec, base_point, n_steps=N_STEPS):
         base_point = gs.to_ndarray(base_point, to_ndim=2)
         tangent_vec = gs.to_ndarray(tangent_vec, to_ndim=2)
 
+        n_base_points = base_point.shape[0]
+        n_tangent_vecs = tangent_vec.shape[0]
+        if n_base_points < n_tangent_vecs:
+            base_point = gs.tile(base_point, (n_tangent_vecs, 1))
+        elif n_base_points > n_tangent_vecs:
+            tangent_vec = gs.tile(tangent_vec, (n_base_points, 1))
```

Comment:
This case is not tested. Does it make sense though, as usually one vector is not tangent at two different points, except in this case of an open set in R^n...?

Reply:
I hadn't thought of it but you're right, it doesn't make much sense to consider one vector and several base points. Maybe just remove this possibility ? Then just allow ```n_base_points = n_tangent_vecs``` and ```n_tangent_vecs > n_base_points = 1```.

Reply:
Yes, let's remove this possibility and add a check that `n_base_points == n_tangent_vecs or n_tangent_vecs > n_base_points == 1`